tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -54,6 +54,7 @@
     ['window-close-v2', SawyerWindowCloseV2Policy(), 0., 0.98, {}],
     ['window-close-v2', SawyerWindowCloseV2Policy(), .1, 0.97, {}],
     ['button-press-v1', SawyerButtonPressV1Policy(), 0., 0.94, {}],
+    ['shelf-place-v2', SawyerShelfPlaceV2Policy(), 0.1, 0.93, {}],
```

Comment:
Can you add a line to the no noise test list as well? It’s just nice to have a record of success rates with and without noise.